docs/reproject.rst

```diff
@@ -0,0 +1,56 @@
+.. _reproject:
+
+=======================
+Reprojecting ND Objects
+=======================
+
+Reprojecting allows you to transform your ND Objects to use coordinates described by another WCS object.
+Users can use this feature to resample the resolution of ND Objects by providing an appropriate target WCS. It also enables them to remove tiny differences between similar `~ndcube.NDCube` objects to get them onto the same grid.
+
+.. _cube_reproject:
+
+Reprojecting an NDCube
+======================
+
+Reprojecting returns a new `~ndcube.NDCube` object that has been transformed to use the provided ``target_wcs``.
+The ``target_wcs`` must be compatible with the WCS that is already associated with your `~ndcube.NDCube`. This means that it should represent the same physical axes and in the same order.
```

Comment:
Mostly you have done this but rst files should have one sentence per line:

```suggestion
The ``target_wcs`` must be compatible with the WCS that is already associated with your `~ndcube.NDCube`. 
This means that it should represent the same physical axes and in the same order.
```

Reply:
Ah okay